projects/extension/src/containers/Options.tsx

```diff
@@ -113,25 +113,23 @@ const Options: React.FunctionComponent = () => {
         desc = "Error"
         break
       case 2:
-        color = "#f99602"
+        color = "#fde047"
         desc = "Warn"
         break
       case 3:
-        color = "#000"
-        desc = "Info"
-        break
       case 4:
-        color = "#5e5e5e"
-        desc = "Debug"
+        color = "#fff"
+        desc = "Info/Debug"
```

Comment:
Info and Debug shouldn't be merged into one, there's a reason why they're two different levels
Debug logs could have a color of `#eee` for example